plugins/modules/win_dns_client.ps1

```diff
@@ -379,6 +423,25 @@ Try {
         }
     }
 
+}
+Catch {
+    $excep = $_
+
+    Write-DebugLog "Exception: $($excep | Out-String)"
+
+    Throw
+}
+
+Try {
+    if (-not (Test-DnsSuffixSearchList $suffix_search_list)) {
+        $result.changed = $true
+        if (-not $check_mode) {
+            Set-DnsClientSuffixSearchList $suffix_search_list
+        }
+        else {
+            Write-DebugLog "Check mode, skipping"
```

Comment:
Would be nice to add something to this string saying `skipping setting suffix_search_list` or something along those lines.